Devscord.DiscordFramework/WorkflowBuilder.cs

```diff
@@ -17,12 +17,21 @@ public class WorkflowBuilder
     {
         public static List<DateTime> ConnectedTimes => Server.ConnectedTimes;
         public static List<DateTime> DisconnectedTimes => Server.DisconnectedTimes;
+        public event Func<Cacheable<IUserMessage, ulong>, ISocketMessageChannel, SocketReaction, Task> ReactionAdded
+        {
+            add { _client.ReactionAdded += value; }
+            remove { _client.ReactionAdded -= value; }
+        }
+        public event Func<Cacheable<IUserMessage, ulong>, ISocketMessageChannel, SocketReaction, Task> ReactionRemoved
+        {
+            add { _client.ReactionRemoved += value; }
+            remove { _client.ReactionRemoved -= value; }
+        }
```

Comment:
zobacz jak to wygląda w przypadku innych eventów, np. AddOnUserJoinedHandlers

Reply:
A mogę zrobić to podobnie jak w przypadku np: SetDefaultMiddlewares? Prościej chyba będzie.

Reply:
to powinno być ustawiane z automatu pod spodem 
tak jak przy obsłudze przychodzących wiadomości

Reply:
to znaczy tak czy nie? Ja tych handlerów to nie ogarniam, a jakbym zrobił na podobnie do powyższych przykładów byłoby łatwiej.